packages/polaris-viz-core/src/constants.ts

```diff
@@ -11,6 +11,7 @@ export const SMALL_CHART_HEIGHT = 125;
 export const FONT_SIZE = 11;
 export const TOUCH_FONT_SIZE = 12;
 
+export const FONT_WEIGHT = 300;
```

Comment:
This font weight is used in the shared `SingleTextLine` component thats used across many components for labels and such. Perhaps `LABEL_FONT_WEIGHT` would be more descriptive? or maybe we just define this default in the component itself, leaning towards the latter 🤔

Reply:
If it's only actually being used for this chart, even if it's consumed in a more generic component, I think it would make sense for this to be defined in the component